modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs

```diff
@@ -97,6 +97,11 @@ public static class LegacyCliConfigKeys {
         /// </summary>
         public const string MaxMessageSize = "MaxMessageSize";
 
+        /// <summary>
+        /// Key for the max (IoT Hub D2C) messages
+        /// </summary>
+        public const string MaxOutgressMessages = "maxoutgressmessages";
```

Comment:
All other keys have camel-case, please change `MaxOutgressMessages` similarly.

Reply:
updated to use camel-case